site/layouts/partials/sidebar.html

```diff
@@ -1,4 +1,4 @@
-<div id="sidebar">
+<div id="sidebar" style="overflow: scroll">
```

Comment:
Let's only do this on y axis

Reply:
Also we should do this in the stylesheet

Reply:
@BenTheElder 
Hi Benjamin, i updated in https://github.com/kubernetes-sigs/kind/pull/1703/commits/dfb524f2f3f6344679e27c1d701c8e717e3ada2a, PTAL again.